impl/src/test/groovy/io/jsonwebtoken/impl/lang/DefaultCollectionMutatorTest.groovy

```diff
@@ -128,6 +132,35 @@ class DefaultCollectionMutatorTest {
         assertEquals 0, changeCount
     }
 
+    @Test
+    void replace() {
```

Comment:
Assert the contents of the collection as well, not just the number of changes/size.

Reply:
I had

> `assertEquals 80, ((MacAlgorithm) m.getCollection().toArray()[0]).getKeyBitLength()`

to check that the original `MacAlgorithm` (with `minKeyBitLength`=512) is indeed replaced with the custom  `MacAlgorithm` (with `minKeyBitLength`=80).